lib/signing/nuget/sign.sh

```diff
@@ -21,12 +21,13 @@ fi
 found=false
 for nuget_package_path in $(find dotnet -name *.nupkg -not -iname *.symbols.nupkg); do
   found=true
-  echo "🔑 Applying authenticode signatures to assemblies in ${nuget_package_path}"
-  for file in $(unzip -Z1 ${nuget_package_path} '*.dll'); do
+  nuget_package=$(cd $(dirname ${nuget_package_path}) && echo $PWD)/$(basename ${nuget_package_path})
```

Comment:
It looks like what you're trying to do is make this path absolute.

Most Linuxes will have the tool `realpath` to do this (won't exist on Mac by default though), and according to StackOverflow you can use `readlink -f <file>` as well.

Reply:
Noted - thanks for explaining that and showing me an alternative!